tests/system/env.yml

```diff
@@ -15,3 +15,6 @@ V3IO_PASSWORD:
 
 # access key used to add v3io mounts to pods - e.g. 4c1c2011-618f-434a-abd6-d456770fd33c
 V3IO_ACCESS_KEY:
+
+# spark service name for spark ingestion
+DEFAULT_SPARK_SERVICE:
```

Comment:
Generally I suggest everything to be prefixed with `MLRUN_` (and have a matching config value) - 
All the `V3IO_` ones were not prefixed because they are already kind of "standards" - so unless there is some standard for spark as well, I suggest changing it to be `MLRUN_FUNCTION_DEFAULTS__SPARK__SERVICE_NAME` (and you'll need to add it in `config.py`)